lua/lvim/core/telescope/custom-finders.lua

```diff
@@ -82,4 +82,15 @@ function M.view_lunarvim_changelog()
   }):find()
 end
 
+-- Smartly opens either git_files or find_files, depending on whether the working directory is
+-- contained in a Git repo.
+function M.find_project_files()
+  local builtin = require "telescope.builtin"
```

Comment:
```suggestion
```

This is what's causing the CI error, since this variable is already defined.

Reply:
Sorry, I'm just looping back to this now. I'll get it fixed.